client/src/components/Todo-List/TodoList.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from "react";
+import Todo from "../Todo/Todo";
+import "../Todo-List/TodoList.scss";
+
+const TodoList = () => {
+	let [value, setValue] = useState("");
+	let [list, setList] = useState([]);
+	useEffect(() => {
+		getItemsFromServer();
+	}, []);
+
+	const getItemsFromServer = async () => {
+		await getItems().then((todos) => {
+			setList(todos);
+		});
+	};
+
+	const getItems = async () => {
+		const response = await fetch("http://localhost:3000/items");
+		const todos = await response.json();
+		return todos;
+	};
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+	};
+
+	const handleTrashClicked = (id) => {
+		deleteItem({ id }).then(() => {
+			getItemsFromServer();
+		});
+	};
+
+	const handleCheckClicked = (id, statusItem) => {
+		const isDone = ({ statusItem } = true);
+		postIsCompleted(id, isDone).then(() => {
+			getItemsFromServer();
+		});
+	};
+
+	const postIsCompleted = async (id, isDone) => {
+		await fetch(`http://localhost:3000/item/${id}`, {
+			method: "POST",
+			headers: { "Content-Type": "application/json" },
+			body: JSON.stringify({ id, isDone }),
+		});
+	};
+
+	const handleClick = async (e) => {
+		const response = await postItem(value);
+		console.log(response);
+		const newItem = await getItemsFromServer();
+		setValue("");
+	};
+
+	// const getItemsFromServer = async () => {
+	// 	await getItems().then((todos) => {
+	// 		setList(todos);
+	// 	});
+	// };
+
+	const handleTextChange = (e) => {
+		setValue(e.target.value);
+	};
+
+	const deleteItem = async (item) => {
+		await fetch("http://localhost:3000/item", {
+			method: "DELETE",
+			headers: { "Content-Type": "application/json" },
+			body: JSON.stringify({ item }),
+		});
+	};
+
+	const postItem = async (item) => {
+		await fetch("http://localhost:3000/item", {
+			method: "POST",
+			headers: { "Content-Type": "application/json" },
+			body: JSON.stringify({ item }),
+		});
+	};
+
+	return (
+		<div>
+			<form onSubmit={handleSubmit}>
+				<input
+					className="todoInput"
+					type="text"
+					value={value}
+					onChange={handleTextChange}
+					placeholder="Enter new to do"
+				/>
+				<button onClick={handleClick} className="submitButton" type="submit">
+					+{" "}
```

Comment:
Don't use real text or concatenation of text and spaces to create buttons, use an icon for "+" and css for spaces (padding / margin)